### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by tiffanyqi**
Update the README with info from Setup for New Devs. Includes:
- What to download
- How to set it up (esp with getting Google keys)
- Running the app
- Issues related to running the app and its solutions
- What we're built on
- Our team
- License

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,187 @@
...
+
+- If you're using bash, add the following as your environment variables in ~/.bashrc
+	```
+	export CJ_DJANGO_SECRET="still-secret"
+	export CJ_GOOGLE_CALENDAR_API_CLIENT_ID="YOUR_CLIENT_ID_HERE"
+	export CJ_GOOGLE_CALENDAR_API_CLIENT_SECRET="YOUR_CLIENT_SECRET_HERE"
+	export APP_ENVIRONMENT="dev"
+	```
+
+- Refresh your command line so that the secret keys are in your environment. You can also run `source ~/.bashrc`. You can check that it work with `echo $APP_ENVIRONMENT`
```
Comment (MaxFangX) [inline README.md]: "Check that it *works with"

File: README.md [hunk 2]
```diff
@@ -1,11 +1,187 @@
...
+
+- Cd into the project directory and run the following:
```
Comment (MaxFangX) [inline README.md]: Make it a code style `cd`, at lowercase at the very least.

File: README.md [hunk 3]
```diff
@@ -1,11 +1,187 @@
...
+	```
+	pip install -r requirements.txt
+	```
+
+- Run the following:
+	```
+	python manage.py bower install
+	```
+	Select "d3#^3.4.4 which resolved to 3.5.17 and is required by nvd3#1.8.5" for the right d3 version
```
Comment (MaxFangX) [inline README.md]: Heads up, this selection process should go away soon after deploy-2 is merged.

File: README.md [hunk 4]
```diff
@@ -1,11 +1,187 @@
...
+```
+python manage.py migrate
+```
+
+Run the app itself by running
+```
+python manage.py runserver
+```
+
+Finally, to live update the front-end when you make a change, run one of the following:
```
Comment (MaxFangX) [inline README.md]: "Finally, to generate `.css` files from your `.scss` changes automatically, run..."

File: README.md [hunk 5]
```diff
@@ -1,11 +1,187 @@
...
+$echo PATH
+```
+
+#### Angular 404 Message
+Try running the following: 
+```
+python manage.py bower install
+```
+
+Make sure you select "d3#^3.4.4 which resolved to 3.5.17 and is required by nvd3#1.8.5"!
```
Comment (MaxFangX) [inline README.md]: Same comment as above. I would remove this.

File: README.md [hunk 6]
```diff
@@ -1,11 +1,187 @@
...
+
+- DO NOT DOWNGRADE to google-api-python-client 1.3.2 (will throw another kind of error)
+
+#### Category20 undefined
+Run the following: 
+```
+python manage.py bower update
+```
+
+Be sure to choose “d3#^3.4.4 which resolved to 3.5.17 and is required by nvd3#1.8.5”!
```
Comment (MaxFangX) [inline README.md]: Remove this

File: README.md [hunk 7]
```diff
@@ -1,11 +1,187 @@
...
+## Our Team
+- [Max Fang](http://maxfangx.com/): Full-Stack Developer, Creator
+- [Tiffany Qi](http://tiffanyqi.com/): Product Manager, Front-End Developer
+- [James Uejio](http://jamesuejio.com/): Full-Stack Developer
+- [Cindy Tung](https://www.linkedin.com/in/cindytung96): Full-Stack Developer
+- [David Nguyen](https://www.linkedin.com/in/dnguyenv): Visual Designer
+- [Andrew Huang](http://andrewhuang.tk): Full-Stack Developer
+
+## License
+The MIT License (MIT)
```
Comment (MaxFangX) [inline README.md:176]: Is there any potential for this to become a legal issue in the event that we want to privatize our code and turn our product into a startup?

**Review by MaxFangX — APPROVED** [review]
+1 Can merge after implementing comments

**Comment by MaxFangX** [comment]
This is fantastic stuff. +1

File: README.md (hunk 3 above)
Comment (MaxFangX) [reply README.md]: Update - remove this line. You can check in [my branch](https://github.com/MaxFangX/calendar-analytics/pull/126) that you don't need this anymore.

File: README.md (hunk 5 above)
Comment (MaxFangX) [reply README.md]: Has been fixed. Please remove.

File: README.md (hunk 7 above)
Comment (tiffanyqi) [reply README.md:176]: If we privatize it, there shouldn't be an issue right?

**Pull request merged**